src/adapters/local.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs')
+const path = require('path')
+
+module.exports = config => {
+
+  let storagePath = config.path;
+  let normalizedPath = path.normalize(`${storagePath}/`)
+
+  return {
+    /** 
+     * copy data into a file on local
+     * @param file          byte array to be saved in a file
+     * @param options.path      relative path for the file to storagePath 
+     * @param options.filename    name of a file to be saved
+     */
+    upload (file, options) {
```

Comment:
minor: could you remove space between function name and '('?

Reply:
Yeap